src/coreclr/dlls/mscoree/mscorwks_unixexports.src

```diff
@@ -126,15 +126,10 @@ mono_gc_set_incremental
 mono_gc_set_max_time_slice_ns
 mono_gc_wbarrier_generic_store
 mono_gc_wbarrier_set_field
-mono_gchandle_free
 mono_gchandle_free_v2
-mono_gchandle_get_target
 mono_gchandle_get_target_v2
```

Comment:
Are we going to be stuck with the `v2` tags indefinitely similar to `BleedingEdge`? It's fine if so, just curious.

Reply:
That's what mono did upstream, so I assume we stick with that forever.